app/scripts/modules/core/instance/instances.component.ts

```diff
@@ -121,6 +136,16 @@ export class InstancesComponent implements IComponentOptions {
   public controller: any = InstancesController;
 }
 
+export class InstancesWrapperComponent implements IComponentOptions {
+  public bindings: any = {
+    instances: '<',
+    highlight: '<'
+  };
+  public controller: any = InstancesController;
+  public template = `<instances instances="$ctrl.instances" highlight="$ctrl.highlight"/>`;
```

Comment:
OH, there was a weird angular bug when trying to load the instances component and I wasn't sure if there was some name collision, so that was an attempt to fix it that I never reverted.